akka-cluster-sharding/src/main/scala/akka/cluster/sharding/Shard.scala

```diff
@@ -226,7 +226,7 @@ private[akka] class Shard(
 
       if (state.entities.nonEmpty) {
         handOffStopper = Some(context.watch(context.actorOf(
-          handOffStopperProps(shardId, replyTo, idByRef.keySet, handOffStopMessage))))
+          handOffStopperProps(shardId, replyTo, idByRef.keySet, handOffStopMessage, settings.tuningParameters.handOffTimeout))))
```

Comment:
It seems *kind* of reasonable to re-use the handOffTimeout setting here, but I do think we should document this.

Reply:
Where should I put the documentation?

Reply:
above the setting in the `reference.conf` seems fine to me - unless someone feels we should introduce a new setting for this? Might be hard to split in a neat backwards-compatible way later.

Reply:
It's done